lib/smart_answer/calculators/business_coronavirus_support_finder_calculator.rb

```diff
@@ -65,11 +64,11 @@ def initialize
       },
       lrsg_open: lambda { |calculator|
         calculator.business_based == "england" &&
-          calculator.restricted_sector == "no"
+          !calculator.sectors.include?("nightclubs_or_adult_entertainment")
```

Comment:
Just a suggestion - would using `exclude?` be better here?  I know `exclude?` is defined in `ActiveSupport` - however I feel it reads better.

Reply:
ahh nice! I'll update that

Reply:
Nice - reads much better 👍